src/app/modules/main/profile_section/notifications/module.nim

```diff
@@ -133,16 +133,17 @@ method addChat*(self: Module, chatDto: ChatDto) =
   let ind = self.view.exemptionsModel().findIndexForItemId(chatDto.id)
   if(ind != -1):
     return
+  echo "Create item"
   let item = self.createChatItem(chatDto)
+  echo "created item"
   self.view.exemptionsModel().addItem(item)
+  echo "added item"
 
 method addChat*(self: Module, itemId: string) =
   let ind = self.view.exemptionsModel().findIndexForItemId(itemId)
   if(ind != -1):
     return
   let chatDto = self.controller.getChatDetails(itemId)
-  if chatDto.chatType != ChatType.OneToOne and chatDto.chatType != ChatType.PrivateGroupChat:
```

Comment:
no idea why this stopped being needed, but who am I to argue?

Reply:
Sorry, I should have added a comment. This function ends up calling the other `addChat`, which also does this same check (see line 131)

Reply:
oh all the other changes are supposed to make this be called only for a community, right?